jobs/ci-kubernetes-bazel-build-1-6.sh

```diff
@@ -34,7 +34,7 @@ if [[ "${rc}" == 0 ]]; then
     echo "Kubernetes version missing; not uploading ci artifacts."
     rc=1
   else
-    bazel run //:ci-artifacts -- "gs://kubernetes-release-dev/bazel/${version}" && rc=$? || rc=$?
+    bazel run //:push-build -- "gs://kubernetes-release-dev/bazel/${version}" && rc=$? || rc=$?
```

Comment:
I'll not change 1-6.sh for now. I don't think it's a big deal if we don't cherrypick.